provision_gke_cluster.sh

```diff
@@ -7,7 +7,7 @@ response=$(gcloud container clusters describe $1 --zone=$2 || echo "ClusterNotFo
 if [[ $response = "ClusterNotFound" ]]
 then
   echo "Creating cluster "$1
-  gcloud container clusters create $1 --num-nodes=1 --machine-type=n1-standard-1
+  gcloud container clusters create $1 --num-nodes=1 --machine-type=n1-standard-1 --zone=$2
```

Comment:
If we're not providing the zone here while creating a cluster it was complaining that cluster zone not defined